### DISCUSSION TITLE
prisma2 hooks for nexus

### DISCUSSION THREAD

**Discussion by darioaxel** [prisma/prisma#2527] [Q&A]
Hi, 
I'm trying to adapt the graphql-auth example in javascript, but as the prisma.yaml file doesn't exist, there is no way to use the hook for nexus when the schema is changed. In the docs hooks are also not mentioned. Could you please help me to learn how to use it?
Thanks!

**Comment by ryands17** [comment]
Are you looking for this in Prisma 1 as there's no `prisma.yaml` file in Prisma 2. Also by hooks do you mean for the schema to get updated automatically on change?

**Comment by ludralph** [comment]
Hi @darioaxel,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
